welcome.php

```diff
@@ -48,7 +49,7 @@ class Welcome extends Module
     public function __construct()
     {
         $this->name = 'welcome';
-        $this->version = '3.0.0';
+        $this->version = '3.0.1';
```

Comment:
You do not need to modify the version by yourself, it is updated when we release a new package of the module. :)

Reply:
thx I removed it